documentation/modules/vmware-prerequisites.adoc

```diff
@@ -15,6 +15,11 @@ The following prerequisites apply to VMware migrations:
 * You must obtain the SHA-1 fingerprint of the vCenter host.
 * If you are migrating more than 10 VMs from an ESXi host in the same migration plan, you must increase the NFC service memory of the host.
 
+[NOTE]
+====
+{project-first} does not support conversion of Btrfs for migrating guest VMs from VMWare.
```

Comment:
I'd also mention that Btrfs is not supported in Openshift Virtualization so that it won't appear as a gap on our side